guides/conquering-the-python-environment.md

```diff
@@ -0,0 +1,4 @@
+# Conquering the Python Environment
+
+**Problem**: You've ran `pip3 ...` but the command doesn't seem to be affecting your Python script.
```

Comment:
"You installed the Aqueduct package via `pip3`, but the package was installed to a different Python environment from what your `python3` is pointing to."